src/accessibility/describe.js

```diff
@@ -80,41 +78,43 @@ p5.prototype.describe = function(text, display) {
   const cnvId = this.canvas.id;
   //calls function that adds punctuation for better screen reading
   text = _descriptionText(text);
-
-  //if it is the first time describe() is called
-  if (!dummy[cnvId + 'fallbackDesc'] || !dummy[cnvId + 'labelDesc']) {
-    //store copy of body dom in dummy
-    _populateDummyDOM(cnvId);
+  //if there is no dummyDOM
+  if (!this.dummyDOM) {
+    this.dummyDOM = document.getElementsByTagName('body')[0];
   }
-
-  //check if text is different
-  if (dummy[cnvId + 'fallbackDesc'] !== text) {
-    //if html structure for description is ready
-    if (dummy[cnvId + 'updateFallbackDesc']) {
+  //check if html structure for description is ready
+  if (this.dummyDOM.querySelector(`#${cnvId}${fallbackDescId}`)) {
```

Comment:
seeing this queries makes me wonder if we gain a little performance by having separate variables for each, like:

this.fallbackDescElement
this.labelDescElement

etc. so the check can just be if the appropriate element exists, and if it doesn't, it gets injected into the DOM. hope that makes sense.